apps/io.gangstergorillaz/manifest.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "Gangster Gorillaz.",
+  "href": "https://gangstergorillaz.io/",
+  "desc": "Gangster Gorillaz NFT. Not afraid to get their hands dirty. 5000 Genesis Gangster Gorillaz Coming to take VeFam on a journey through BlockChain Land. To eventually land in Our "Banana-Verse" All together.",
```

Comment:
should be `'Banana-Verse'`

Reply:
Updated the PR.

Reply:
Sorry @libotony Hope you guys still around 💯